### DISCUSSION TITLE
MPC link_poses?

### DISCUSSION THREAD

**Discussion by Spphire** [NVlabs/curobo#65] [Software Q&A] [answered]
motion_gen.plan_single can set link_poses, can mpc do the same thing?

**Comment by balakumar-s** [comment, marked as answer]
Yes, mpc, ik and motion_gen all can set link_poses. They all call this class to compute costs: https://github.com/NVlabs/curobo/blob/d85ae41fba22672d8178e3bed2af464119936e1b/src/curobo/rollout/arm_reacher.py#L207

**Comment by Spphire** [comment]
thx, I found it =>"links_goal_pose"
